### ISSUE TITLE
Running dev-scripts fails with: ocp_install_env.sh: line 59: cd: /src/github.com/openshift/installer: No such file or directory 

### ISSUE THREAD

**Issue by mcornea** [openshift-metal3/dev-scripts#848]
**Describe the bug**

Running dev-scripts on a freshly installed machine fails with:

```bash
[kni@worker-1 dev-scripts]$ cat logs/03_build_installer-2019-11-18-152310.log 
+ source utils.sh
++ set -o pipefail
+ source common.sh
+++ go env
common.sh: line 3: go: command not found
++ eval ''
++ export KNI_INSTALL_FROM_GIT=true
++ KNI_INSTALL_FROM_GIT=true
++ export PATH=/bin:/usr/local/go/bin:/home/kni/.local/bin:/home/kni/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/sbin:/usr/local/sbin
++ PATH=/bin:/usr/local/go/bin:/home/kni/.local/bin:/home/kni/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/sbin:/usr/local/sbin
++ export GOTRACEBACK=crash
++ GOTRACEBACK=crash
++ export MOBY_DISABLE_PIGZ=true
++ MOBY_DISABLE_PIGZ=true
++++ dirname common.sh
+++ cd .
+++ pwd
++ SCRIPTDIR=/home/kni/dev-scripts
+++ whoami
++ USER=kni
++ '[' -z '' ']'
++ '[' -f /home/kni/dev-scripts/config_kni.sh ']'
++ echo 'Using CONFIG /home/kni/dev-scripts/config_kni.sh'
Using CONFIG /home/kni/dev-scripts/config_kni.sh
++ CONFIG=/home/kni/dev-scripts/config_kni.sh
++ source /home/kni/dev-scripts/config_kni.sh
+++ set +x
+++ BOOTSTRAP_SSH_READY=2500
+++ NODES_PLATFORM=baremetal
+++ INT_IF=eth1
+++ PRO_IF=eth0
+++ CLUSTER_PRO_IF=ens3
+++ EXT_IF=
+++ ROOT_DISK=/dev/sda
+++ NODES_FILE=/home/kni/instackenv.json
+++ MANAGE_BR_BRIDGE=n
+++ NUM_WORKERS=1
+++ CLUSTER_NAME=ocp-edge-cluster
+++ BASE_DOMAIN=qe.lab.redhat.com
+++ DNS_VIP=192.168.123.6
+++ EXTERNAL_SUBNET=192.168.123.0/24
+++ grep -o 'registry.svc.ci.openshift.org[^"]\+'
+++ curl https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.3.0-0.ci/latest
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   239  100   239    0     0    788      0 --:--:-- --:--:-- --:--:--   788
++ LATEST_CI_IMAGE=registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
++ export OPENSHIFT_RELEASE_IMAGE=registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
++ OPENSHIFT_RELEASE_IMAGE=registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
++ export OPENSHIFT_INSTALL_PATH=/src/github.com/openshift/installer
++ OPENSHIFT_INSTALL_PATH=/src/github.com/openshift/installer
++ '[' false '!=' false ']'
++ env
++ grep -q _LOCAL_IMAGE=
++ '[' -z true ']'
++ export OPENSHIFT_INSTALLER=/src/github.com/openshift/installer/bin/openshift-install
++ OPENSHIFT_INSTALLER=/src/github.com/openshift/installer/bin/openshift-install
++ export OPENSHIFT_INSTALLER_RHCOS=file:////src/github.com/openshift/installer/data/data/rhcos.json
++ OPENSHIFT_INSTALLER_RHCOS=file:////src/github.com/openshift/installer/data/data/rhcos.json
++ export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
++ OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
++ env
++ grep -q _LOCAL_IMAGE=
++ ADDN_DNS=
++ EXT_IF=
++ PRO_IF=eth0
++ MANAGE_BR_BRIDGE=n
++ MANAGE_PRO_BRIDGE=y
++ MANAGE_INT_BRIDGE=y
++ INT_IF=eth1
++ ROOT_DISK_NAME=/dev/sda
++ FILESYSTEM=/
++ WORKING_DIR=/opt/dev-scripts
++ NODES_FILE=/home/kni/instackenv.json
++ NODES_PLATFORM=baremetal
++ MASTER_NODES_FILE=ocp/master_nodes.json
++ export METAL3_DEV_ENV=
++ METAL3_DEV_ENV=
++ '[' -z '' ']'
++ export VM_SETUP_PATH=/opt/dev-scripts/metal3-dev-env/vm-setup
++ VM_SETUP_PATH=/opt/dev-scripts/metal3-dev-env/vm-setup
++ export NUM_MASTERS=3
++ NUM_MASTERS=3
++ export NUM_WORKERS=1
++ NUM_WORKERS=1
++ export VM_EXTRADISKS=false
++ VM_EXTRADISKS=false
++ export IRONIC_IMAGE=quay.io/metal3-io/ironic:master
++ IRONIC_IMAGE=quay.io/metal3-io/ironic:master
++ export IRONIC_IPA_DOWNLOADER_IMAGE=quay.io/metal3-io/ironic-ipa-downloader:master
++ IRONIC_IPA_DOWNLOADER_IMAGE=quay.io/metal3-io/ironic-ipa-downloader:master
++ export IRONIC_DATA_DIR=/opt/dev-scripts/ironic
++ IRONIC_DATA_DIR=/opt/dev-scripts/ironic
++ export VBMC_IMAGE=quay.io/metal3-io/vbmc
++ VBMC_IMAGE=quay.io/metal3-io/vbmc
++ export SUSHY_TOOLS_IMAGE=quay.io/metal3-io/sushy-tools
++ SUSHY_TOOLS_IMAGE=quay.io/metal3-io/sushy-tools
++ export KUBECONFIG=/home/kni/dev-scripts/ocp/auth/kubeconfig
++ KUBECONFIG=/home/kni/dev-scripts/ocp/auth/kubeconfig
++ export 'SSH=ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5'
++ SSH='ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5'
++ export LIBVIRT_DEFAULT_URI=qemu:///system
++ LIBVIRT_DEFAULT_URI=qemu:///system
++ '[' kni '!=' root -a /run/user/1000 == /run/user/0 ']'
++ sudo -n uptime
+++ awk -F= '/^ID=/ { print $2 }' /etc/os-release
+++ tr -d '"'
++ [[ ! rhel =~ ^(centos|rhel)$ ]]
+++ awk -F= '/^VERSION_ID=/ { print $2 }' /etc/os-release
+++ tr -d '"'
+++ cut -f1 -d.
++ VER=8
++ [[ 8 -ne 7 ]]
++ [[ 8 -ne 8 ]]
++ grep -q 'Red Hat Enterprise Linux release 8' /etc/redhat-release
++ export RHEL8=True
++ RHEL8=True
+++ df / --output=fstype
+++ grep -v Type
++ FSTYPE=xfs
++ case ${FSTYPE} in
+++ xfs_info /
+++ grep -q ftype=1
++ [[ -n '' ]]
++ '[' 3955 = 0 ']'
++ '[' '!' -d /opt/dev-scripts ']'
++ '[' '!' -d /opt/dev-scripts/ironic ']'
++ export TEST_CUSTOM_MAO=false
++ TEST_CUSTOM_MAO=false
+ source ocp_install_env.sh
+++ go env
++ eval 'GOARCH="amd64"
GOBIN=""
GOCACHE="/home/kni/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/kni/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build206508145=/tmp/go-build -gno-record-gcc-switches"'
+++ GOARCH=amd64
+++ GOBIN=
+++ GOCACHE=/home/kni/.cache/go-build
+++ GOEXE=
+++ GOFLAGS=
+++ GOHOSTARCH=amd64
+++ GOHOSTOS=linux
+++ GOOS=linux
+++ GOPATH=/home/kni/go
+++ GOPROXY=
+++ GORACE=
+++ GOROOT=/usr/local/go
+++ GOTMPDIR=
+++ GOTOOLDIR=/usr/local/go/pkg/tool/linux_amd64
+++ GCCGO=gccgo
+++ CC=gcc
+++ CXX=g++
+++ CGO_ENABLED=1
+++ GOMOD=
+++ CGO_CFLAGS='-g -O2'
+++ CGO_CPPFLAGS=
+++ CGO_CXXFLAGS='-g -O2'
+++ CGO_FFLAGS='-g -O2'
+++ CGO_LDFLAGS='-g -O2'
+++ PKG_CONFIG=pkg-config
+++ GOGCCFLAGS='-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build206508145=/tmp/go-build -gno-record-gcc-switches'
++ export BASE_DOMAIN=qe.lab.redhat.com
++ BASE_DOMAIN=qe.lab.redhat.com
++ export CLUSTER_NAME=ocp-edge-cluster
++ CLUSTER_NAME=ocp-edge-cluster
++ export CLUSTER_DOMAIN=ocp-edge-cluster.qe.lab.redhat.com
++ CLUSTER_DOMAIN=ocp-edge-cluster.qe.lab.redhat.com
+++ cat /home/kni/.ssh/id_rsa.pub
++ export 'SSH_PUB_KEY=ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDyNMDlGcwRqqkObTA4KJSw2+I/hM3j37Dt+fGfobHzf0/Wgu/9NJmuPKC9tCgblrBdsJhsmZkKX1W/A0Ned69mhFGkPYSV4LZuX1y93gB3Rmu6514pikQ8/OjQfjBU9XaVuVLIj+AUEE/bpdPMcO9iZxuT5v5MiS6gOvXago91jFHFuB+KcjC1yVKGaH+X4DJh0uE4TckTHZbFjHa6uCKYMORyXA2RkMIQKuEgrais2WPjtlFFTQ7OpJqCt6UVXV+pbdx5p6JdyHz/FGR0gvcCkeNJhHoFg/3WH6ik0dODShcp3RWZKy82tj7zvOCsq1j5MDGYqqogImz8ZqC28chV kni@worker-1.example.com'
++ SSH_PUB_KEY='ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDyNMDlGcwRqqkObTA4KJSw2+I/hM3j37Dt+fGfobHzf0/Wgu/9NJmuPKC9tCgblrBdsJhsmZkKX1W/A0Ned69mhFGkPYSV4LZuX1y93gB3Rmu6514pikQ8/OjQfjBU9XaVuVLIj+AUEE/bpdPMcO9iZxuT5v5MiS6gOvXago91jFHFuB+KcjC1yVKGaH+X4DJh0uE4TckTHZbFjHa6uCKYMORyXA2RkMIQKuEgrais2WPjtlFFTQ7OpJqCt6UVXV+pbdx5p6JdyHz/FGR0gvcCkeNJhHoFg/3WH6ik0dODShcp3RWZKy82tj7zvOCsq1j5MDGYqqogImz8ZqC28chV kni@worker-1.example.com'
++ export EXTERNAL_SUBNET=192.168.123.0/24
++ EXTERNAL_SUBNET=192.168.123.0/24
++ export DNS_VIP=192.168.123.6
++ DNS_VIP=192.168.123.6
+ extract_oc registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
++ mktemp -d installer--XXXXXXXXXX
+ extract_dir=installer--ElICIyMqjr
+ extract_command oc registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359 installer--ElICIyMqjr
+ local release_image
+ local cmd
+ local outdir
+ local extract_dir
+ cmd=oc
+ release_image=registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
+ outdir=installer--ElICIyMqjr
++ mktemp -d installer--XXXXXXXXXX
+ extract_dir=installer--xDegZidyOL
++ mktemp pullsecret--XXXXXXXXXX
+ pullsecret_file=pullsecret--X0YbhjD8NN
+ echo '{ "auths": { "cloud.openshift.com":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K21jb3JuZWFyZWRoYXRjb20xZmM0MHZzZ3h3cWVxa294dmpxeWk0eHF0ZXk6V0YxU0U2Rk82NlYzSE5CMVczUVFDOE9CSk81MFJGTFVNRVZPTUhUNEE2WVQ5MlVRRFJTTUZOVEQyNjYzRUQ3Vg==","email":"mcornea@redhat.com"},"quay.io":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K21jb3JuZWFyZWRoYXRjb20xZmM0MHZzZ3h3cWVxa294dmpxeWk0eHF0ZXk6V0YxU0U2Rk82NlYzSE5CMVczUVFDOE9CSk81MFJGTFVNRVZPTUhUNEE2WVQ5MlVRRFJTTUZOVEQyNjYzRUQ3Vg==","email":"mcornea@redhat.com"},"registry.connect.redhat.com":{"auth":"NzQ4NDMyN3x1aGMtMUZDNDBWc0d4V3FFcWtPeFZKUXlJNFhRVGV5OmV5SmhiR2NpT2lKU1V6VXhNaUo5LmV5SnpkV0lpT2lJd01UWTRPRGhrTXpreE9UTTBOakpoWVdZNU1XUXdNalkwT0RCa01qWXhPU0o5LlBUVWlRcDRkMUozMW9maGpacm1zRjUxUDN5RGFZSzdOZC1FR0pxV0hHc0FIS0F0UlZ4ajZuUHl6LUlHb2RBdV8zUWdCX3hZRkN4NFVsZ3hheFhtVkNpU2k1NENvaVFyRTBkRlk5eHFxWUhwcXFTVlprS21LWGNPQ1NXRjF2OExRRTBJQVVEZm8xZlRtQ09KYUxsVnlVTnI3ZlFrYkdSRC1ZdWRhNEY2cFBZVHNlcnJnS1JsekRBRXNIdjJ4Rm9VcGJJYzVZb0czSzJuOVlBSFBhNHVVb25fckhDQTJadmZQb3V1ajdtZlZ2MjEtallXdGU0NTV5X1pZcjlfX095RElLSHdabjFqQ3Q5VG5IVC0yRFlQRl9WeThYek1GWTEwSmlqTmVHZEhmQkJ1RUtadFk5eS1SRjJ4QTlSNU9YcVNrdDczODRxWDd3VHBjWUh3MjlJeXVJREI2c0ZJaDFTWkdHTHpaZGJPUDBibzViMXV3YjRyTDhLRDlobld1SENkOVY0eC1RYjBQYlNYVDhRZEZSNVZCeG1BYkpGWHY5cU9lckRmckZWU0ZsVDByaVE0SmZKdVNUdUdYQWhwV25UREtyanJjYUpocmtsa09CSUx2YVNGVElCYl9QRkJxQmJaRnJmTkF0WHJZOU9DcmRZcEJWdVRwc01uaG84SWtyX09DSUhwRnFHdFZ4dkotRkkzeWtoOHEzWUt2aUNwd3VlVHhGY1JlOTdpMUxiYzlLV1JMWVhoNEVjVEtKenA0TjQ5WUM2aDRZM2JVZHpCeEU4bFVVQU55ZmlhWXFPa0pCUWRaOXNGb3VUa3JwU2xfUUFZN1l5LXpSNkxxZ3Z6QjF6UlFtemZZM2VtTmVYbHhXTjZtYUlpZ09NOFRmZm5ndlVldE45dl9tMDc4bmFv","email":"mcornea@redhat.com"},"registry.redhat.io":{"auth":"NzQ4NDMyN3x1aGMtMUZDNDBWc0d4V3FFcWtPeFZKUXlJNFhRVGV5OmV5SmhiR2NpT2lKU1V6VXhNaUo5LmV5SnpkV0lpT2lJd01UWTRPRGhrTXpreE9UTTBOakpoWVdZNU1XUXdNalkwT0RCa01qWXhPU0o5LlBUVWlRcDRkMUozMW9maGpacm1zRjUxUDN5RGFZSzdOZC1FR0pxV0hHc0FIS0F0UlZ4ajZuUHl6LUlHb2RBdV8zUWdCX3hZRkN4NFVsZ3hheFhtVkNpU2k1NENvaVFyRTBkRlk5eHFxWUhwcXFTVlprS21LWGNPQ1NXRjF2OExRRTBJQVVEZm8xZlRtQ09KYUxsVnlVTnI3ZlFrYkdSRC1ZdWRhNEY2cFBZVHNlcnJnS1JsekRBRXNIdjJ4Rm9VcGJJYzVZb0czSzJuOVlBSFBhNHVVb25fckhDQTJadmZQb3V1ajdtZlZ2MjEtallXdGU0NTV5X1pZcjlfX095RElLSHdabjFqQ3Q5VG5IVC0yRFlQRl9WeThYek1GWTEwSmlqTmVHZEhmQkJ1RUtadFk5eS1SRjJ4QTlSNU9YcVNrdDczODRxWDd3VHBjWUh3MjlJeXVJREI2c0ZJaDFTWkdHTHpaZGJPUDBibzViMXV3YjRyTDhLRDlobld1SENkOVY0eC1RYjBQYlNYVDhRZEZSNVZCeG1BYkpGWHY5cU9lckRmckZWU0ZsVDByaVE0SmZKdVNUdUdYQWhwV25UREtyanJjYUpocmtsa09CSUx2YVNGVElCYl9QRkJxQmJaRnJmTkF0WHJZOU9DcmRZcEJWdVRwc01uaG84SWtyX09DSUhwRnFHdFZ4dkotRkkzeWtoOHEzWUt2aUNwd3VlVHhGY1JlOTdpMUxiYzlLV1JMWVhoNEVjVEtKenA0TjQ5WUM2aDRZM2JVZHpCeEU4bFVVQU55ZmlhWXFPa0pCUWRaOXNGb3VUa3JwU2xfUUFZN1l5LXpSNkxxZ3Z6QjF6UlFtemZZM2VtTmVYbHhXTjZtYUlpZ09NOFRmZm5ndlVldE45dl9tMDc4bmFv","email":"mcornea@redhat.com"},"registry.svc.ci.openshift.org": { "auth": "c3lzdGVtLXNlcnZpY2VhY2NvdW50LWtuaS1kZWZhdWx0OmV5SmhiR2NpT2lKU1V6STFOaUlzSW10cFpDSTZJaUo5LmV5SnBjM01pT2lKcmRXSmxjbTVsZEdWekwzTmxjblpwWTJWaFkyTnZkVzUwSWl3aWEzVmlaWEp1WlhSbGN5NXBieTl6WlhKMmFXTmxZV05qYjNWdWRDOXVZVzFsYzNCaFkyVWlPaUpyYm1raUxDSnJkV0psY201bGRHVnpMbWx2TDNObGNuWnBZMlZoWTJOdmRXNTBMM05sWTNKbGRDNXVZVzFsSWpvaVpHVm1ZWFZzZEMxMGIydGxiaTAxZEdkbU55SXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMbTVoYldVaU9pSmtaV1poZFd4MElpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WlhKMmFXTmxMV0ZqWTI5MWJuUXVkV2xrSWpvaVlqZzNNRGt4WmpZdE5qRXlNeTB4TVdVNUxXRTJNVGt0TkRJd01UQmhPR1V3TURBeUlpd2ljM1ZpSWpvaWMzbHpkR1Z0T25ObGNuWnBZMlZoWTJOdmRXNTBPbXR1YVRwa1pXWmhkV3gwSW4wLm51VGR0RlczRENHcFpvT0pCbU45VjQwWG1wbmlZRE9tUnI2Z05vNGVwRVBrb1lDXzk1YmhWX0ttYjhoTnprOTNVTGtDNnJXNTVjTXFQMVM4RHh3QWw0RUxRZ2NFZXIyalBJLXZBNGUzdlZ5cHNLbS1XSkFxcWo2OGhNN0Z4ekMzRGgxY19lN19EQkJLOWtxZmcyRzZiNTJXQmI2RUhsODg2Q2Nza3JBVm1fbmprNS14ay1Ma1hSM3lXNW5JeXlZdXhNVGg1LUNMd3lQQy1yLVIzeklzdnlWelNPVTgyeUJaaE1tUmc3enUtOWlydThENHdqRFJQclhiSm1FV3lBM1FIUlJ2VTJuci01MTFEeEhEbWhtNW14YU0tSFA4emk3SU8zVEU5SU55S3BqTmo5eTIwNmtFN0NNSVNMWmRWWFl3MkpIQ1BmSmJQMHNJY3V0dnFvOTdGdw==" } } }'
+ oc adm release extract --registry-config pullsecret--X0YbhjD8NN --command=oc --to installer--xDegZidyOL registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-11-18-100359
+ mv installer--xDegZidyOL/oc installer--ElICIyMqjr
+ rm -rf installer--xDegZidyOL
+ rm -rf pullsecret--X0YbhjD8NN
+ sudo mv installer--ElICIyMqjr/oc /usr/local/bin
+ rm -rf installer--ElICIyMqjr
+ mkdir -p ocp/
+ '[' -z true ']'
+ clone_installer
+ [[ ! -d /src/github.com/openshift/installer ]]
+ sync_repo_and_patch go/src/github.com/openshift/installer https://github.com/openshift/installer.git https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2657.patch https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2666.patch https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2673.patch
+ REPO_PATH=/home/kni
+ DEST=/home/kni/go/src/github.com/openshift/installer
+ echo 'Syncing go/src/github.com/openshift/installer'
Syncing go/src/github.com/openshift/installer
+ '[' '!' -d /home/kni/go/src/github.com/openshift/installer ']'
+ mkdir -p /home/kni/go/src/github.com/openshift/installer
+ git clone https://github.com/openshift/installer.git /home/kni/go/src/github.com/openshift/installer
Cloning into '/home/kni/go/src/github.com/openshift/installer'...
Checking out files: 100% (15754/15754), done.
+ pushd /home/kni/go/src/github.com/openshift/installer
~/go/src/github.com/openshift/installer ~/dev-scripts
+ git am --abort
fatal: Resolve operation not in progress, we are not resuming.
+ true
+ git checkout master
Already on 'master'
Your branch is up to date with 'origin/master'.
+ git fetch origin
+ git rebase origin/master
Current branch master is up to date.
+ test 5 -gt 2
+ git branch -D metalkube
error: branch 'metalkube' not found.
+ true
+ git checkout -b metalkube
Switched to a new branch 'metalkube'
+ shift
+ shift
+ for arg in "$@"
+ git am
+ curl -L https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2657.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 15102    0 15102    0     0  51193      0 --:--:-- --:--:-- --:--:-- 51193
Applying: Cache uncompressed images, as installers can't actually use compressed images.
+ for arg in "$@"
+ curl -L https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2666.patch
+ git am
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10338    0 10338    0     0  60104      0 --:--:-- --:--:-- --:--:-- 60104
Applying: RHCOS: Bump to 43.81.201911131833.0 for CRI-O bug fix
+ for arg in "$@"
+ curl -L https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2673.patch
+ git am
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1240    0  1240    0     0   8157      0 --:--:-- --:--:-- --:--:--  8157
Applying: data/baremetal/bootstrap: increase disk size to 16GiB
+ popd
~/dev-scripts
+ build_installer
+ pushd .
~/dev-scripts ~/dev-scripts
+ cd /src/github.com/openshift/installer
ocp_install_env.sh: line 59: cd: /src/github.com/openshift/installer: No such file or directory

```

**Comment by yprokule** [comment]
should be addressed by #844

**Comment by mcornea** [comment]
> should be addressed by #844

Thanks!
